lib/granite/action/policies.rb

```diff
@@ -33,23 +33,26 @@ module Policies
       end
 
       module ClassMethods
-        # The simplies policy. Takes block and executes it returning
-        # boolean result. Multiple policies are reduced with ||
+        # The simplest policy. Takes either a symbol or a block and executes it
+        # returning boolean result. Multiple policies are reduced with ||
         #
         #   class Action < Granite::Action
         #     allow_if { performer.is_a?(Recruiter) }
         #     allow_if { performer.is_a?(AdvancedRecruiter) }
+        #     allow_if :staff?
         #   end
         #
-        # The first argument in block is a current action performer,
-        # so it is possible to use a short-cut performer methods:
+        # If symbol is passed, an instance method with the same name is executed.
+        # If block is passed, the first argument of the block is the current action
```

Comment:
Wondering why we pass `perfomer` as an argument to the block, it seems that it's available in that context anyway.

Reply:
To allow `&:staff?` type of passing. Would work without having an argument